plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-order-summary-block/edit.tsx

```diff
@@ -16,14 +21,37 @@ import {
 	getAllowedBlocks,
 } from '../../../cart-checkout-shared';
 import { OrderMetaSlotFill } from './slotfills';
+import { useContainerWidthContext } from '../../../../base/context';
+import { FormattedMonetaryAmount } from '../../../../../../packages/components';
```

Comment:
We can get this from `@woocommerce/blocks-components`  in most cases. (Unless the file is also in `packages/components`)

Reply:
Whoops, yep I often forget to convert these, since the editor auto-imports I have to go back later and change everything to the aliases. Not a big fan of the aliases as they just add work with no value except it "looks tidy". 🤷🏻

Reply:
Fixed https://github.com/woocommerce/woocommerce/pull/52253/commits/e2148c22cd2e6f50e64b45ccc5711b8e39f4f8d5